pkgs/development/interpreters/python/mk-python-derivation.nix

```diff
@@ -144,7 +144,7 @@ in
 
 , meta ? {}
 
-, doCheck ? config.doCheckByDefault or false
+, doCheck ? true
```

Comment:
It's a bit late now, but I think this fix was not the one that needed to happen, since `doCheck` never gets used.
Only `attrs.doCheck` is ever referenced, which doesn't need this line.
```suggestion
```
Should we open another PR that just removes this line?

Reply:
Ah, yes. Removing this line from the set pattern is a better solution to such confusion.